react/components/RenderProvider.tsx

```diff
@@ -192,6 +184,13 @@ class RenderProvider extends Component<Props, RenderProviderState> {
     }
   }
 
+  public joinTreePath = (treePath: string, id: string) => {
+    const { runtime: {runtimeMeta: {pagesProtocol = 1}} } = this.props
+    const separator = pagesProtocol >= 2 ? ' ' : '/'
+
+    return [treePath, id].filter(each => !!each).join(separator)
```

Comment:
Why do we have this `each => !!each`?

Reply:
I'm worried about it because some of the functions refactored to this one didn't apply this filter...